pkg/reconciler/revision/table_test.go

```diff
@@ -906,6 +908,13 @@ func allUnknownConditions(r *v1.Revision) {
 	MarkActivating("Deploying", "")(r)
 }
 
+// Revision Unknown conditions but with the generated default PA conditions
+// taken into consideration
+func withRevisionConditionsGivenPADefault(r *v1.Revision) {
+	WithInitRevConditions(r)
+	r.Status.MarkActiveUnknown("Deploying", "")
+}
+
```

Comment:
This looks identical to `allUnknownConditions` can we clean up the extra function and simplify the diff?

https://github.com/nader-ziada/serving/blob/f1341d21581b613facc929cb3ed0bad197eafd53/pkg/reconciler/revision/table_test.go#L905-L909